google-cloud-pubsub/lib/google/cloud/pubsub.rb

```diff
@@ -290,12 +289,18 @@ module Cloud
     # pubsub = Google::Cloud::Pubsub.new
     #
     # sub = pubsub.subscription "my-topic-sub"
-    # received_message = sub.pull.first
-    # if received_message
+    # subscriber = sub.listen do |received_message|
```

Comment:
Why is this param different from other block params for listen? `received_message` may be more readable versus `msg`, but it's understandable either way. I'm asking more for consistency.

Reply:
I also thought about changing all existing examples to `received_message` for consistency, and would be happy to do so if @blowmage is OK with the change.

Reply:
I'm okay with either `msg` or `received_message` _or_ `rec_msg`...

Reply:
Let's do received_message. I appreciate the verbosity.

Reply:
OK, I will update all `listen` and `pull` examples to `received_message`.

Reply:
@frankyn @blowmage Updated. (There is one example in which I had to use `rec_message` due to line length.)